package.json

```diff
@@ -8,11 +8,11 @@
   "private": true,
   "scripts": {
     "dr:surge": "node dr-surge.js",
-    "build": "npm run clean && NODE_ENV=production $(npm bin)/webpack --config ./config/webpack.prod.js",
+    "build": "npm run clean && NODE_ENV=production BRAND_TYPE=Konveyor $(npm bin)/webpack --config ./config/webpack.prod.js",
```

Comment:
I think this would override an exported BRAND_TYPE from outside this line, which I think the container builds use (`BRAND_TYPE=RedHat npm run build`). If we need a default we should probably just fall back on Konveyor when we read the variable if it's undefined.